### ISSUE TITLE
Navbar button not aligned to title when on --prod build [ionic-angular 3.7.x]

### ISSUE THREAD

**Issue by polcham** [ionic-team/ionic#13197]
**Ionic version:**
[ ] **2.x**
[X] **3.x**
[ ] **4.x**

**I'm submitting a ...**
[X] bug report
[ ] feature request

**Current behavior:**

The Navbar button is not aligned to the Navbar title when on a --prod build.
[Screenshot](https://i.imgur.com/0UGYCfM.png)

This is how it looks when deploying without --prod flag
[Screenshot](https://i.imgur.com/7h4YzEI.png)

The behavior is only happening on ionic-angular 3.7.x, downgrading to ionic-angular 3.6.x properly builds a --prod build.

**Expected behavior:**

The Navbar button should be aligned to the Navbar title.
[Screenshot](https://i.imgur.com/7h4YzEI.png)

**Steps to reproduce:**

1. Start a new Ionic 3 app
2. Add a Navbar with button
3. Compile, build, and run with --prod flag

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):
```
cli packages: (/Users/paulcham/Repositories/fireline/node_modules)

    @ionic/cli-utils  : 1.13.1
    ionic (Ionic CLI) : 3.13.2

global packages:

    cordova (Cordova CLI) : 7.0.1

local packages:

    @ionic/app-scripts : 3.0.0
    Cordova Platforms  : android 6.2.3 ios 4.4.0
    Ionic Framework    : ionic-angular 3.7.1

System:

    ios-deploy : 1.9.2
    Node       : v8.6.0
    npm        : 5.5.1
    OS         : macOS Sierra
    Xcode      : Xcode 8.3.3 Build version 8E3004b

Misc:

    backend : legacy
```

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us!

There should be a new release of `@ionic/app-scripts` in the near future that addresses this issue. If you would like to test now, you can apply the nightly build as follows:

```
npm install @ionic/app-scripts@3.0.0-201710161925 --save --save-exact
```

I will let you know when the new version is released.

Thank you for using Ionic

**Comment by kensodemann** [comment]
OK, so scratch that last bit. The latest `@ionic/app-scripts` was just released, so please try this and let me know if it works:

```
npm install @ionic/app-scripts@latest --save --save-exact
```

This should result in version `3.0.1`

Then re-build using `--prod` and let me know the results.

**Comment by Sroose** [comment]
Thanks @kensodemann, we had sthe same issue and it was indeed solved by @ionic/app-scripts 3.0.1

**Comment by kensodemann** [comment]
Sweet!  I will close this assuming it will also fix the issue for the OP. If not, I can always reopen it... :)
